data/sql_incremental_aggregates/prepare_schedule_a_aggregate_size.sql

```diff
@@ -33,6 +33,10 @@ create index on ofec_sched_a_aggregate_size_tmp (size);
 create index on ofec_sched_a_aggregate_size_tmp (total);
 create index on ofec_sched_a_aggregate_size_tmp (count);
 
+-- this drops totals during rebuild
+drop table if exists ofec_sched_a_aggregate_state cascade;
+drop table if exists ofec_sched_a_aggregate_size_old cascade;
```

Comment:
I added these statements so that you could do update_all more than once. If there is currently an aggregate_size_old table, it needs to dropped because it is two iterations ago, then the current aggregate_size table becomes the new aggregate_size_old table. 

If I am understanding you right, you saying that the cascade will get rid of the old old table which will get rid of the current table. (I was not sure of that dependency) In that case, renaming the current table is not helpful.